make/autoconf/flags-cflags.m4

```diff
@@ -629,7 +629,7 @@ AC_DEFUN([FLAGS_SETUP_CFLAGS_HELPER],
   STATIC_LIBS_CFLAGS="-DSTATIC_BUILD=1"
   if test "x$TOOLCHAIN_TYPE" = xgcc || test "x$TOOLCHAIN_TYPE" = xclang; then
     STATIC_LIBS_CFLAGS="$STATIC_LIBS_CFLAGS -ffunction-sections -fdata-sections \
-      -DJNIEXPORT='__attribute__((visibility(\"hidden\")))'"
+      -DJNIEXPORT='[[gnu::visibility(\"hidden\")]]'"
```

Comment:
So IIUC we now use attributes via the C++11 syntax rather than compiler-specific syntax - even where the C++11 syntax is referring to a compiler specific attribute. Is that right?

Reply:
Yep, just something that C++ does a little neater, at least in my view

Reply:
We (the HotSpot Group) have not discussed or approved the use of the new C++ attribute syntax, whether for standard attributes or compiler-specific ones.  That involves an update to the Style Guide.  I'm not convinced switching existing uses from compiler-specific `__attribute__` syntax to compiler-specific `[[attribute]]` syntax is worth the substantial code churn.

Reply:
I was under the assumption that anything not listed under "Not discussed" or "Forbidden" was free game in HotSpot, I apologise for my goof since that isn't the case. That said, I am curious if there are any isolated conversations about the attribute syntax anywhere

Reply:
@kimbarrett Aside from the attribute syntax (which you're already doing in a new proposal), now that `alignas` has been approved for use in HotSpot, is there anything that can be done for [8250269:Replace ATTRIBUTE_ALIGNED with alignas](https://bugs.openjdk.org/browse/JDK-8250269) and the related Pull Request (https://git.openjdk.org/jdk/pull/11431)?